raccine-reg-patch-pshell.reg

```diff
@@ -0,0 +1,4 @@
+Windows Registry Editor Version 5.00

```

Comment:
Please rename the file to `raccine-reg-patch-powershell.reg`

Reply:
Also done.